src/Model.elm

```diff
@@ -524,7 +525,7 @@ processMastodonEvent msg model =
             case result of
                 Ok context ->
                     { model | currentView = ThreadView (Thread status context) }
-                        ! [ Command.scrollColumnToBottom "thread" ]
+                        ! [ Ports.scrollIntoView <| "thread-status-" ++ (toString status.id) ]
```

Comment:
How about moving this to a dedicated fn in the `Command` module? That's what have been done so far, it's kinda convenient to know all commands are issued from this specific module.